platform/logging/LoggingConfig.ts

```diff
@@ -0,0 +1,166 @@
+import { Schema, typeOfSchema } from '../types';
+import { LogLevelId } from './LogLevel';
+import { Appenders, AppenderConfigType } from './appenders/Appenders';
+
+/**
+ * Separator string that used within nested context name (eg. plugins::pid).
+ */
+const CONTEXT_SEPARATOR = '::';
```

Comment:
Yep, I'll add comment in the PR and file separate issue that I'll be handling next as it's the major pain point.